src/components/Directory/Footer.tsx

```diff
@@ -0,0 +1,111 @@
+import React from "react";
+
+const Footer = () => {
+  return (
+    <section>
+      <div className="fr-container--fluid fr-py-7w fr-bg--tilleul-light">
+        <div className="fr-container">
+          <h2>A retenir</h2>
+          <div className="fr-grid-row fr-grid-row--gutters">
+            <div className="fr-col-12 fr-col-md-4">
+              <h3 className="fr-text--dark-blue">
+                Avant le RDV avec le psychologue
+              </h3>
+              <ul className="no-bullet">
+                <li>
+                  <strong>
+                    <span
+                      className="fr-fi-arrow-right-line fr-fi--md fr-text--dark-blue"
+                      aria-hidden="true"
+                    />{" "}
+                    Vous prenez rendez-vous avec un médecin{" "}
+                  </strong>{" "}
+                  (généraliste, pédiatre, gériatre, spécialiste, médecin
+                  scolaire, de PMI, des services de santé des universités ou
+                  encore d’un médecin hospitalier).
+                </li>
+                <li>
+                  <strong>
+                    <span
+                      className="fr-fi-arrow-right-line fr-fi--md fr-text--dark-blue"
+                      aria-hidden="true"
+                    />
+                  </strong>{" "}
+                  En cas d’imprévu, pensez à bien prévenir le psychologue de
+                  votre désistement. En informant rapidement de votre absence,
+                  vous permettez à un autre patient de bénéficier d’un suivi.
+                </li>
+              </ul>
+            </div>
+            <div className="fr-col-12 fr-col-md-4">
+              <h3 className="fr-text--dark-blue">
+                Le jour du RDV avec le psychologue
+              </h3>
+              <ul className="no-bullet">
+                <li>
+                  <span
+                    className="fr-fi-arrow-right-line fr-fi--md fr-text--dark-blue"
+                    aria-hidden="true"
+                  />{" "}
+                  <strong> Vous apportez le courrier d’adressage</strong> rédigé
+                  par le médecin.
+                </li>
+
+                <li>
+                  <span
+                    className="fr-fi-arrow-right-line fr-fi--md fr-text--dark-blue"
+                    aria-hidden="true"
+                  />{" "}
+                  Si vous n’avancez pas les frais en raison de soins en lien
+                  avec une maladie, une maternité ou un AT-MP, vous apportez
```

Comment:
```suggestion
                  avec une maladie, une maternité ou un AT-MP, vous apportez{" "}
```

Et aussi, je crois qu'on essayait d'éviter les acronymes (AT-MP), à vérifier.